.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - "0.10"
```

Comment:
we don't need to have multiple Node versions, we're not testing anything on Node, maybe the only issue we might have is due to dependencies, but we don't use anything else than devDeps so far.